memo.rb

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'csv'
+require 'securerandom'
+require 'sinatra'
+require 'sinatra/reloader'
+
+HEADERS = %w[
+  id
+  title
+  text
+].freeze
+
+MEMOS_CSV = 'memos.csv'
```

Comment:
```suggestion
CSV_PATH = 'memos.csv'
```

みたいに抽象度を上げた方がいいですね（具体的すぎると変更に弱くなる）

https://bootcamp.fjord.jp/articles/78